LAKMobile/src/components/ImagePickerButton.tsx

```diff
@@ -2,21 +2,29 @@ import React from 'react';
 import { Image, Pressable, StyleSheet } from 'react-native';
 import Icon from 'react-native-vector-icons/MaterialCommunityIcons';
 import { AppText } from './AppText';
+import { COLORS } from '../../constants';
 
 interface ImagePickerButtonProps {
   sourceURI?: string;
   onSelect: () => void;
+  isDisabled: boolean;
```

Comment:
I think it's more concise if we have it just be `disabled` instead of `isDisabled` since they both mean the same thing. Additionally, we can have `disabled?: boolean`, since `undefined` is a falsey value and will evaluate to false if disabled isn't specified. This means that we don't need to add a disabled field every time we create an ImagePickerButton